### PR TITLE
fix(TabView): `disableSwipe` & `disableTransition` prop 

### PR DESCRIPTION
**Pull request by Sinharitik589**
**What kind of change does this PR introduce?**

feature

**Did you add tests for your changes?**
NO

**If relevant, did you update the documentation?**
NO

**Summary**

#3232 

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

File: packages/base/src/TabView/TabView.tsx [hunk 1]
```diff
@@ -33,6 +33,9 @@ export type TabViewBaseProps = {
 
   /** Styling for TabView.Item Component container. */
   tabItemContainerStyle?: StyleProp<ViewStyle>;
+
+  /** Swipe enabled or not */
+  swipeEnabled?: Boolean;
```
Comment (arpitBhalla) [inline packages/base/src/TabView/TabView.tsx:38]: `swipeEnabled`->`disableSwipe`

File: packages/base/src/TabView/TabView.tsx [hunk 2]
```diff
@@ -91,37 +95,72 @@ export const TabViewBase: RneFunctionComponent<TabViewBaseProps> = ({
...
+        <Animated.View
+          testID="tabView-test"
+          style={StyleSheet.flatten([
+            styles.container,
             {
-              translateX: translateX.interpolate({
-                inputRange: [0, 1],
-                outputRange: [0, -window.width],
-              }),
+              width: window.width * length,
```
Comment (arpitBhalla) [inline packages/base/src/TabView/TabView.tsx:105]: Don't repeat same code

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

File: packages/base/src/TabView/TabView.tsx [hunk 3]
```diff
@@ -66,7 +70,9 @@ export const TabViewBase: RneFunctionComponent<TabViewBaseProps> = ({
     }
     const position = dx / -window.width;
     const next = position > value ? Math.ceil(position) : Math.floor(position);
-    onChange?.(currentIndex.current + next);
+    if (!disableSwipe) {
+      onChange?.(currentIndex.current + next);
+    }
```
Comment (arpitBhalla) [inline packages/base/src/TabView/TabView.tsx:75]: I don't think this would make any change, did you tried on example app?

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

**Comment by arpitBhalla** [comment]
Thank you for your contribution 🎉

**Comment by monk-lee** [comment]
Can you let me use it
`react-native-elements/react-native-elements#base`
`react-native-elements/react-native-elements#themed`, too?

**Comment by arpitBhalla** [comment]
yeah, just delete lock file and reinstall

**Pull request merged**